.github/workflows/preview-pr.yml

```diff
@@ -0,0 +1,74 @@
+#
+# Build and preview documentation in PRs
+# DOES NOT WORK FOR FORKS.
+name: Preview PR
+
+on:
+  pull_request:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.0
+
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
```

Comment:
You need to add submodule option herr

Reply:
Done in https://github.com/gmt-china/gmt-china.org/pull/179/commits/fd6d20d60df5d431860251e3497d52677d4598ed